src/app/docs-pages/loading-spinner-doc/loading-spinner-doc.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="doc-page-container">
+
+  <h1 class="doc-title">Loading spinner component</h1>
+
+  <p class="doc-info">
+    This is the documentation for the hal-loading-spinner component. The loading-spinner can be used in use cases where users have to wait until
+    further interactions are available.
+  </p>
+
+  <hal-doc-import-line>
+    <code><pre>import {{ '{' }} LoadingSpinnerModule {{ '}' }} from '@hafslundnett/hal-components';</pre></code>
+  </hal-doc-import-line>
+
+  <h2 class="doc-subtitle">Examples</h2>
+<div class="loading-spinner-container">
+  <hal-loading-spinner></hal-loading-spinner>
+</div>
+
+  <hal-code-highlighter [isHTML]="true" [code]="htmlCode"></hal-code-highlighter>
+
+    <p class="doc-info">
+      The loading spinner comes defualt in color '--hdd-blue', 
+      and does not have any other inputs that to call the selector tag 'hal-loading-spinner' to create it. 
```

Comment:
Kanskje skrive om denne: and does not have any other inputs that to call the selector tag 'hal-loading-spinner' to create it.

Reply:
Endret den nå =)